.travis.yml

```diff
@@ -1,13 +1,12 @@
 language: dart
 
 dart:
-  - 2.3.0
-  - stable 
+  - dev
 
 dart_task:
-  - test: -p vm
-  - test: -p chrome
-  - dartanalyzer: --fatal-warnings --fatal-infos .
+  - test: --enable-experiment=non-nullable -p vm
```

Comment:
I don't think this works - the dart_task test configuration doesn't allow vm arguments (this would be a test executable argument).

You actually don't need this though for this package (assuming it is in the allow list)